libs/utils/android/workload.py

```diff
@@ -134,4 +134,31 @@ def tracingStop(self):
         # Dump a platform description
         self._te.platform_dump(self.out_dir)
 
+    def traceShow(self):
+        """
+        Open the collected trace using the most appropriate native viewer.
+
+        The native viewer depends on the specified trace format:
+        - ftrace: open using kernelshark
+        - systrace: open using a browser
+
+        In both cases the native viewer is assumed to be available in the host
+        machine.
+        """
+
+        if 'ftrace' in self.collect:
+            os.popen("kernelshark {}".format(self.trace_file))
+            return
+
+        if 'systrace' in self.collect:
```

Comment:
Good catch, it also seems to be part of the standard libraries.
I cannot check on other platforms but at least it's working on my machine.
Will update and switch to use this API.